src/koopmans/workflows/_singlepoint.py

```diff
@@ -132,8 +133,10 @@ def _run(self) -> None:
                     if all(self.atoms.pbc) and self.calculator_parameters['ui'].do_smooth_interpolation:
                         if not Path('pkipz/postproc').is_dir():
                             utils.system_call('mkdir pkipz/postproc')
-                        for dir in ['wannier', 'TMP', 'pdos']:
+                        for dir in ['wannier', 'TMP']:
                             utils.system_call(f'rsync -a ki/postproc/{dir} pkipz/postproc/')
+                        if any([isinstance(c, ProjwfcCalculator) for c in self.calculations]):
```

Comment:
Would it not be simpler and more future-proof to include `pdos` in the list of directories to copy, and then just have a check to see if the directory exists before attempting the `rsync`?

Reply:
I agree!